plugins/bazaar/src/components/AddProjectDialog/AddProjectDialog.tsx

```diff
@@ -65,6 +66,7 @@ export const AddProjectDialog = ({
     reset: UseFormReset<FormValues>,
   ) => {
     const formValues = getValues();
+    formValues.name = createSlugFromTitle(formValues.title);
```

Comment:
So this is still required, is it? But now the client side code auto-generates a name from the title.